astropy/units/format/cds.py

```diff
@@ -60,8 +61,9 @@ class CDS(Base, _ParsingFormatMixin):
         "DIMENSIONLESS",
     )
 
-    @classproperty(lazy=True)
-    def _units(cls) -> dict[str, UnitBase]:
+    @classmethod
+    @cache
```

Comment:
total nit, but  I think the reverse order looks better for these decorators. Does order matter ?

Reply:
Yes, the order matters.